### PR TITLE
feature: add pouch port cmd

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

### Ⅰ. Describe what this PR did
support `pouch port` cmd.

### Ⅱ. Does this pull request fix one issue?

fix part of #238

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
I have added it.

### CODE REVIEW

File: test/cli_container_port_test.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"github.com/alibaba/pouch/test/command"
+	"github.com/alibaba/pouch/test/environment"
+	"github.com/go-check/check"
```
Comment (ZYecho) [inline test/cli_container_port_test.go]: add a blank line upon this line.

File: test/cli_container_port_test.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+// SetUpSuite does common setup in the beginning of each test suite.
+func (suite *PouchContainerPortSuite) SetUpSuite(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+	PullImage(c, busyboxImage)
+}
+
+// Test pouch port
+func (suite *PouchContainerCopySuite) TestPouchPort(c *check.C) {
```
Comment (ZYecho) [inline test/cli_container_port_test.go]: PouchContainerCopySuite -> PouchContainerPortSuite?

File: test/cli_container_port_test.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	command.PouchRun("run",
+		"--name", name, "-d",
+		"-p", testcase1,
+		"-p", testcase2,
+		"-p", testcase3,
+		busyboxImage,
+		"sh", "-c", "sleep 10000").Assert(c, icmd.Success)
+	defer DelContainerForceMultyTime(c, name)
+
+	ret := command.PouchRun("port",
```
Comment (ZYecho) [inline test/cli_container_port_test.go]: how about add a test case when len(arg) = 1

File: test/cli_container_cp_test.go [hunk 4]
```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/gotestyourself/gotestyourself/icmd"
 )
 
-// APIContainerCopySuite is the test suite for container cp CLI.
+// PouchContainerCopySuite is the test suite for container cp CLI.
```
Comment (ZYecho) [inline test/cli_container_cp_test.go]: good catch!

File: test/cli_container_port_test.go (hunk 1 above)
Comment (yeya24) [reply test/cli_container_port_test.go]: Done.

File: test/cli_container_port_test.go (hunk 2 above)
Comment (yeya24) [reply test/cli_container_port_test.go]: Done. Thanks.

File: test/cli_container_port_test.go (hunk 3 above)
Comment (yeya24) [reply test/cli_container_port_test.go]: Done.

File: test/cli_container_cp_test.go (hunk 4 above)
Comment (yeya24) [reply test/cli_container_cp_test.go]: :smile: Can you please review this PR again?

File: test/cli_container_port_test.go [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+		name, "5000").Assert(c, icmd.Success)
+	c.Assert(ret.Stdout(), check.Equals, "127.0.0.1:5000\n")
+
+	ret = command.PouchRun("port",
+		name, "8000").Assert(c, icmd.Success)
+	c.Assert(ret.Stdout(), check.Equals, "0.0.0.0:8000\n")
+
+	// Test for only one arg.
```
Comment (ZYecho) [inline test/cli_container_port_test.go]: I think only one arg just like `pouch port 179` where 179 is the container name or ID

Comment (yeya24) [reply test/cli_container_port_test.go]: Sorry for that miss. I have move this comment below. PTAL

**Comment by allencloud** [comment]
LGTM

**Pull request merged**